src/platform/errors.ts

```diff
@@ -10,10 +11,19 @@ export const translateError = (err: Error | number | string): string => {
 	if (err instanceof InternalRequestError) {
 		return 'Server Error';
 	}
-	if (err instanceof Error) {
-		return err.message;
+	let message;
+	if (err instanceof HttpError) {
+		captureException(
+			new Error(),
+			'Translating a HttpError, this should go via handleHttpErrors instead',
```

Comment:
```suggestion
			'Translating an HttpError, this should go via handleHttpErrors instead',
```

Reply:
It depends on your accent @dfunckt, if you say `haitch` it's `a`, if you say `aitch` it's `an` - I'd use `a`

Reply:
Okay, I think I'll trust you on this :)

Reply:
Yeah it really comes down to accent, it's `an` before vowel sounds and `a` before non-vowel sounds, and the letter `H` can be said with either so the correct one depends on the reader.  You can also get vowels with non-vowel sounds like `a ukulele`